src/forwardproblem.cpp

```diff
@@ -86,22 +86,34 @@ void ForwardProblem::workForwardProblem() {
     } catch (...) {
         throw AmiException("AMICI setup failed due to an unknown error");
     }
-    /* initialise objective function values */
+
     if(edata){
-        rdata->llh = 0.0;
-        rdata->chi2 = 0.0;
-        std::fill(rdata->sllh.begin(),rdata->sllh.end(), 0.0);
-        std::fill(rdata->s2llh.begin(),rdata->s2llh.end(), 0.0);
+        rdata->initializeObjectiveFunction();
     }
     
     int ncheck = 0; /* the number of (internal) checkpoints stored so far */
     realtype tlastroot = 0; /* storage for last found root */
 
     /* if preequilibration is necessary, start Newton solver */
     if (solver->getNewtonPreequilibration()) {
+        // Are there dedicated preequilibration parameters provided?
+        if(edata && edata->fixedParametersPreequilibration.size()) {
+            warnMsgIdAndTxt("asdf", "sefsdf");
```

Comment:
i guess we can remove those warning messages now?

Reply:
Sorry, yes, should not have gotten in here.